### PR TITLE
Remove ntp sync reboot

### PR DESCRIPTION
**Pull request by davidwagnerkc**
## Description
Reboot after NTP sync is no longer necessary with the changes to the skills process startup. 

## How to test
Boot up device with no network connection or identity file. Set date back: `sudo date -s "Sep 23 20:40:13 UTC 2019"` and continue to setup wifi, pair device and confirm it works as expected. 

## Contributor license agreement signed?
CLA [Yes ] (Whether you have signed a [CLA - Contributor Licensing Agreement](https://mycroft.ai/cla/)

### CODE REVIEW

File: mycroft/skills/__main__.py [hunk 1]
```diff
@@ -81,20 +81,17 @@ def prepare_device(self):
         """Internet dependent updates of various aspects of the device."""
         self._get_pairing_status()
         self._update_system_clock()
-        if self.time_skew_threshold_exceeded:
-            self._reboot()
-        else:
-            self._update_system()
-            # Above will block during update process and kill this instance if
-            # new software is installed
+        self._update_system()
```
Comment (chrisveilleux) [inline mycroft/skills/__main__.py]: You got rid of the only usage of self.time_skew_threshold_exceeded.  Removal of that property should be included in this PR.

Comment (davidwagnerkc) [reply mycroft/skills/__main__.py]: Thanks. `_reboot()` was also no longer needed and remove. Re-tested and works just fine.

**Comment by penrods** [comment]
I understand why you'd want to get rid of this, but keep an eye on it.  I don't recall the exact symptoms, but it involved an image which was months old being booted for the first time.  In the startup sequence, the sudden passage of those "months" in one clock tick had some squirrelly effects on the system.  This was a little ham-handed, but it was the most reliable solution at the time and seemed reasonable during that case.  Perhaps the one-hour threshold was too aggressive -- I was trying to make it deal reasonably with the unplug, move and replug scenario.

Not saying you should avoid doing this, but be sure to test the impact of a really long restart.  On the  Raspberry Pi they use a thing called [fake-hwclock](https://manpages.debian.org/jessie/fake-hwclock/fake-hwclock.8.en.html) to save the current time in a file at shutdown, then reset the clock on boot.  You can potentially edit that file to simulate a long dormancy -- look for /etc/fake-hwclock.data.

**Comment by davidwagnerkc** [comment]
Thanks I was just testing with setting the date manually back a day. I can
try months and try it with that file rather than the date command as well.

**Comment by davidwagnerkc** [comment]
Did more tests with changing clock back a year and also tested with fresh Mark 1 image that was from June, removing ntp reboot and adding in monotonic time for pairing code expiration (https://github.com/MycroftAI/skill-pairing/commit/2e42197125d5ca87e7d71a5174983135f6db6302). All test continue to look good. Merging.

**Pull request merged**